client/keys/show.go

```diff
@@ -48,7 +48,7 @@ consisting of all the keys provided by name and multisig threshold.`,
 	cmd.Flags().String(FlagBechPrefix, sdk.PrefixAccount, "The Bech32 prefix encoding for a key (acc|val|cons)")
 	cmd.Flags().BoolP(FlagAddress, "a", false, "Output the address only (overrides --output)")
 	cmd.Flags().BoolP(FlagPublicKey, "p", false, "Output the public key only (overrides --output)")
-	cmd.Flags().BoolP(FlagDevice, "d", false, "Output the address in the device")
+	cmd.Flags().BoolP(FlagDevice, "l", false, "Output the address in a ledger device")
```

Comment:
This technically makes the PR "api-breaking", should we reflect that in the pending log?

Reply:
we can actually revert that

Reply:
Let's revert it then 👍